src/Uno.Templates/content/unoapp/.vscode/extensions.json

```diff
@@ -1,8 +1,8 @@
 {
-	"recommendations": [
-		"unoplatform.vscode"
-	],
-	"unwantedRecommendations": [
-		"ms-dotnettools.csdevkit"
-	],
-}
\ No newline at end of file
+  "recommendations": [
+    "unoplatform.vscode"
+  ],
+  "unwantedRecommendations": [
+    "ms-dotnettools.csdevkit"
```

Comment:
We'll need to remember to remove this once dev kit has been fixed. Should prob create a linked issue for this